src/components/atoms/ImageComponent/Image.jsx

```diff
@@ -0,0 +1,21 @@
+import PropTypes from 'prop-types';
+import './Image.scss';
+
+// TODO Add loading on image onLoad
+
+export const ImageComponent = memo(({ src, size, onClick }) => {
+
+    const imageOnClick = useCallback(() => onClick(), [onClick]);
+
+    return (
+
+        <img src={`${src}`} clasName={`atom__image atom_image_${size}`} onClick={imageOnClick}></img>
+
+    );
+});
+
+ImageComponent.propTypes = {
+    size: PropTypes.oneOf(['medium', 'large']),
+    src: img.propTypes.src.isRequired,
```

Comment:
I think that is not a valid type, that should only be a required string, also default props are missing

Reply:
Also there is not such an object as `img`
Please at least run/build the project, do not push code without even checking it